### ISSUE TITLE
/** autocompletion and indentation

### ISSUE THREAD

**Issue by pronebird** [joelday/vscode-docthis#59]
Don't seem to document method that has indentation:

```js
export default class Apple extends Model {

  /** <--- DOES NOTHING
  doSomething() {
    ...
  }

}
```

Starting `/**` off the beginning of line adds comment without issues.

**Comment by jamietre** [comment]
Not sure if this is the same, but `/**` actually doesn't work at all for me on VS Code Insider. It works fine for the current release version. So something they changed recently in VS Code itself seems to have broken auto doc generation from `/**`, Ctrl+Shift+D still works.

**Comment by joelday** [comment]
Yuck. It's time to reevaluate the implementation strategy of this feature most likely.

**Comment by jaydp17** [comment]
Same issue on v1.9.0-insider

**Comment by ashleahhill** [comment]
I'm on v 1.8.1 on Mac and the auto doc generation is broken for me as well

**Comment by edDimensi** [comment]
Have some problem too. 
```
shell.ts:541 Illegal argument: character must be positive: Error: Illegal argument: character must be positive
    at Object.h [as illegalArgument] (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:4:29440)
    at new e (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:9:465)
    at C:\Users\Dimen\.vscode\extensions\joelday.docthis-0.3.6\out\src\extension.js:88:38
    at e.invoke (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:4:31992)
    at e.fire (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:5:8945)
    at t.$acceptModelChanged (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:11:686)
    at t.e.handle (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:10:4203)
    at s (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:7:11178)
    at p (c:\Program Files (x86)\Microsoft VS Code\resources\app\out\vs\workbench\node\extensionHostProcess.js:7:11855)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by joelday** [comment]
@edDimensi Thanks for providing the call stack!

**Comment by Elarcis** [comment]
Yep, issue's still there in 1.9 release. As this is the main method which I use to document, I was surprised this didn't work anymore :D

**Comment by ThomasMichon** [comment]
This feature has been broken for me for a couple months at this point. Typing `/**`, even without indentation, is still failing. Hitting Enter merely triggers VS Code's normal auto-completion for this action.

**Comment by joelday** [comment]
Fixed in 0.3.11. Thank you!
